website/docs/d/satellite_host_script.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+
+subcategory: "Satellite"
+layout: "ibm"
+page_title: "IBM : satellite_attach_host_script"
+description: |-
+  Generate host script to attach host to satellite location.
+---
+
+# ibm\_satellite_host
+
+Import the details of an existing IBM satellite location registration script as a data source. Create a script to run on a Red Hat Enterprise Linux 7 or AWS EC2 host in your on-premises infrastructure. The script attaches the host to your IBM Cloud Satellite location. The host must have access to the public network in order for the script to complete.
+
+## Example Usage
+
+###  Create satellite host script to attach IBM host to satellite control plane
+
+```hcl
+data "ibm_satellite_attach_host_script" "script" {
+  location          = "satellite-ibm"
```

Comment:
"satellite-ibm" use the interpolation syntax

Reply:
Fixed